config/application.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
```

Comment:
@chiraggshah I have disabled Rubocop for `config` and it's subfolders because either ways configs will get modified during upgrade process with Rails. Didn't think Rubocop will add any value additions for `config` folder.

Lemme know your thoughts on the same.

Reply:
Have updated the rules such that config folder is under Rubocop scope but it won't enforce frozen string literal rule for `config` and sub-directories. Frozen string literal gets removed by Rails under config folder whenever we upgrade. And I don't think adding this comment for `config` folder doesn't add much value.